invoicer_project/items/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from django.core.validators import RegexValidator
+from user.models import User
+
+string_validation = RegexValidator(
+    regex=r'^[A-Z][a-z]+',
+    message="Item name has to be entered in the format: 'Apple'."
+)
+price_validation = RegexValidator(
+    regex=r'^\d+([., ]?(\d){1,2}?)?$',
+    message="Price has to be entered in the format: '25.55'."
+    )
+currency_validation = RegexValidator(
+    regex=r'^[A-Z]+$',
+    message="Currency has to be entered in the format: 'UAH'."
+)
+basic_unit_validation = RegexValidator(
+    regex=r'^[a-zA-Z]*$',
+    message="Basic unit has to be entered in the format: 'kg'."
+)
+barcode_validation = RegexValidator(
+    regex=r'^\d{7}$',
+    message="Barcode has to be entered in the format: '1234567'."
+)
+
+class Item(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name='items')
+    name = models.CharField(max_length=35, validators=[string_validation])
+    price = models.FloatField(validators=[price_validation])
+    currency = models.CharField(max_length=7, validators=[currency_validation])
```

Comment:
Why not to use list of predefined currency? It is very inconvenient to write in the field proper currency. Anyway, why length for this field is 7?

Reply:
Good idea! We'll discuss and fix it.

Reply:
@maxchv , on the frontend we have dropdown list to choose currency(chosen value is stored in hidden input and will be send to backend as a string), so we don't write it. Is it possible to do without predefined list of currency in the model and keep field currency as charfield?

Reply:
(will be solved on the add_item page branch)